scipy-1.0/paper.tex

```diff
@@ -1263,6 +1262,48 @@ \section*{Supporting Information}
 \label{fig:sparse-conv}
 \end{figure}
 
+\section{Polynomial Interpolators: Additional Details}
```

Comment:
This section may be a little awkward with various bits and pieces pasted in from the main text now, but maybe not such a big deal now that it has been moved to supporting info. Assuming Evgeni is "ok" with this reduction in length for main text section.